install.sh

```diff
@@ -24,7 +24,7 @@ install_dependencies=false
 install_library=false
 install_prefix="${ROCM_PATH}"
 msccl_kernel_enabled=true
-mscclpp_enabled=true
+mscclpp_enabled=false
```

Comment:
This is fine temporarily, but it actually makes it _impossible_ to enable MSCCL++ through the script, so devs should be aware of this.